functests/performance/updating_profile.go

```diff
@@ -133,6 +133,29 @@ var _ = Describe("[rfe_id:28761] Updating parameters in performance profile", fu
 				}
 			}
 		})
+
+		It("[test_id:22764] verify that realTimeKernerl is default when realTimeKernel field does not exist", func() {
+			timeout := time.Duration(len(workerRTNodes) * 10)
+
+			profile.Spec.RealTimeKernel = nil
+
+			By("Applying changes in performance profile")
+			Expect(testclient.Client.Update(context.TODO(), profile)).ToNot(HaveOccurred())
+
+			if profile.Spec.RealTimeKernel == nil {
```

Comment:
I would make this an expectation: if it's not nil, it's a test failure, otherwise go on